dist/scss/wds-variables/_dropdowns.scss

```diff
@@ -0,0 +1 @@
+$wds-dropdown-content-z-index: 1;
```

Comment:
Please once again rethink scope of those variables. Do we really want encourage people to use this variables?
Can we find a valid use-case for it?